testing/xfpga/test_buffer_c.cpp

```diff
@@ -68,7 +68,7 @@ extern "C" {
 #define FPGA_MSG(fmt, ...) \
 	printf("MOCK " fmt "\n", ## __VA_ARGS__)
 
-
+#pragma pack(1)
```

Comment:
Wouldn't this set this alignment for the rest of the structs in the file?
When setting alignment on structs, it's best to do it explicitly/individually for each struct you want aligned a certain way.
The better approach would be:
```C
#pragma pack(push, 1)
 struct buffer_params {
   fpga_result result;	   fpga_result result;
   size_t size;	   size_t size;
   int flags;	   int flags;
 };
#pragma pack(pop)
```

Reply:
Learning_list.append("#pragma pack(push, 1)
 struct buffer_params {
   fpga_result result;	   fpga_result result;
   size_t size;	   size_t size;
   int flags;	   int flags;
 };
#pragma pack(pop)")